api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatientExporter.java

```diff
@@ -0,0 +1,18 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+import org.openmrs.module.registrationcore.api.mpi.openempi.OpenEmpiPatientQuery;
+
+/**
+ * This service perform export patient to Mpi server.
+ */
+public interface MpiPatientExporter {
+
+    /**
+     * Perform export patient to MPI server
+     *
+     * @param patient patient which should be exported.
+     * @return exported MPI patient
+     */
+    OpenEmpiPatientQuery exportPatient(Patient patient);
```

Comment:
There is a mismatch here between this interface trying to be generic, but the return type of this method returning something OpenEMPI-specific.